architecture/service_catalog/index.adoc

```diff
@@ -87,6 +87,17 @@ This infrastructure allows a loose coupling between applications running in
 those services to focus on its own business logic while leaving the management
 of these services to the provider.
 
```

Comment:
We need a section ID before the heading:
[[service-catalog-deleting-resources]]